terraform/shared/modules/base/variables.tf

```diff
@@ -0,0 +1,26 @@
+variable "cf_org_name" {
```

Comment:
Add a note here indicating that defaults are production defaults, and they're exposed as variables to enable environments to specify alternate config when needed. Reference the production module as the "no config necessary" invocation.